.travis.yml

```diff
@@ -18,7 +18,7 @@ before_script:
   - make
   - sudo make -C fstests setup
 script:
-  - go test ./pkg/...
+  - go test ./...
```

Comment:
This will run the tests in fstests/xxx/ (goflock and gofslock) by accidents, should we exclude them? Or just add `go test ./cmd/...` here

Reply:
Makes sense to me.